connect_transformations/lookup_ff_request/utils.py

```diff
@@ -119,3 +127,19 @@ def validate_lookup_ff_request(data):  # noqa: CCR001
     return {
         'overview': overview,
     }
+
+
+def filter_requests_with_changes(requests):
+    """ FF requests without any changes can be produced during synchronization.
+        Exclude them.
+    """
+    filtered_requests = []
+    for request in requests:
+        changed = False
+        for item_data in request['asset']['items']:
+            if item_data['quantity'] != item_data['old_quantity']:
```

Comment:
I'm not sure that this will work for 100% of scenarios. Can it be 2 request found and both are SYNC? If yes, we will get an empty result.

Reply:
Yes, this is theoretically possible. Thanks!